packages/lit-element/src/lib/decorators/base.ts

```diff
@@ -13,7 +13,6 @@
  */
 
 export type Constructor<T> = {
-  // tslint:disable-next-line:no-any
```

Comment:
`no-any` was already set as warning in ESLint config.
But the `lit-element` package was using TSLint.

Reply:
Ah I see. I don't think we've been consistently running lint anyway, since it's not in our CI yet, so this may not be very intentional. I would suggest keeping it an error, and restoring the exceptions for now. If we do turn on lint enforcement, I think introducing new `any`s should be an error.

@justinfagnani any thoughts on this?

Reply:
And I imagine that the ideal would be to try to eliminate all the anys at some point (maybe better in other PRs), but as you prefer. 🙂